flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/avro/AvroParquetRecordFormat.java

```diff
@@ -50,11 +51,12 @@
 
     private final TypeInformation<E> type;
 
-    private final GenericData dataModel;
+    private final SerializableSupplier<GenericData> dataModelSupplier;
 
-    AvroParquetRecordFormat(TypeInformation<E> type, GenericData dataModel) {
```

Comment:
These changes should be in the previous commit.

Reply:
sure